cmd/gossamer/main.go

```diff
@@ -61,9 +65,26 @@ func main() {
 	}
 }
 
+func AddVerbosity(logger log.Logger, ctx *cli.Context) error {
+	lvl, err := log.LvlFromString(ctx.String("verbosity"))
```

Comment:
This only handles level names not numerical levels: https://github.com/ChainSafe/log15/blob/b92444566a9f836e6ff88bba74cbb98b96e203d6/logger.go#L50

If you have an integer then you can cast it to a log.Lvl (alias for an int): https://github.com/ChainSafe/log15/blob/b92444566a9f836e6ff88bba74cbb98b96e203d6/logger.go#L16